tools/sdk-cli/Cargo.toml

```diff
@@ -16,4 +16,4 @@ serde_json = "1.0"
 tiny-bip39 = "1"
 tokio = { version = "1", features = ["rt-multi-thread"] }
 serde = { version = "1.0", features = ["derive"] }
-clap = { version = "4", features = ["derive"] }
\ No newline at end of file
+clap = { version = "<4.2.*", features = ["derive"] }
```

Comment:
Why not update the local rust version instead?

```
rustup update stable
```

Reply:
This was reported by Ademar in a group chat, he did ask if it's OK to upgrade and didn't get answers so I assumed we were targeting 1.69.0 as a requirement.
I will update my rust version 👍

Reply:
Reverted e48902088783449e148bba6c6b89528c0fa21451